src/modules/endpoint/sagas/watchFetchNews.js

```diff
@@ -0,0 +1,206 @@
+// @flow
+
+import type { Saga } from 'redux-saga'
+import { all, call, put, select, takeLatest } from 'redux-saga/effects'
+import type {
+  FetchNewsActionType,
+  PushNewsActionType,
+  FetchNewsFailedActionType,
+  ClearNewsActionType,
+  FetchMoreNewsActionType
+} from '../../app/StoreActionType'
+import type { DataContainer } from '../DataContainer'
+import { ContentLoadCriterion } from '../ContentLoadCriterion'
+import isPeekingRoute from '../selectors/isPeekingRoute'
+import ErrorCodes, { fromError } from '../../error/ErrorCodes'
+import { LOCAL } from '../../../routes/news/containers/NewsContainer'
+import loadLocalNews from './loadLocalNews'
```

Comment:
`TuNews` is used as `Tunews` everywhere else so please lets change it here as well